server/cache.go

```diff
@@ -354,7 +354,7 @@ func newClusterInfo(id IDAllocator) *clusterInfo {
 		id:              id,
 		stores:          newStoresInfo(),
 		regions:         newRegionsInfo(),
-		writeStatistics: cache.NewDefaultCache(writeStatCacheMaxLen),
+		writeStatistics: cache.NewCache(writeStatCacheMaxLen, cache.TwoQueueCache),
```

Comment:
we have an outer strategy to evicted cold region in the hot write region cache, maybe I need to do some test for this change.

Reply:
any performance improved?

Reply:
@siddontang  LRU may remove a warmer page to make space for the cold one. if the number of hot regions suddenly increase more than the buffer size, although we have an outer strategy, it may remove a warmer page.I test in our environment with smaller size LRU, the 2Q performance better.and I write a simple simulation test like https://play.golang.org/p/AhO0a7O4FR. 
the result:
```
LRU Hit:92.70, 2Q Hit:92.70
LRU Hit:92.20, 2Q Hit:92.20
LRU Hit:93.10, 2Q Hit:93.10
LRU Hit:93.50, 2Q Hit:93.50
LRU Hit:92.20, 2Q Hit:92.20
LRU Hit:91.00, 2Q Hit:91.00
LRU Hit:0.00, 2Q Hit:12.10
LRU Hit:0.00, 2Q Hit:11.10
LRU Hit:0.00, 2Q Hit:11.20
LRU Hit:12.50, 2Q Hit:12.50
LRU Hit:13.10, 2Q Hit:13.10
LRU Hit:92.10, 2Q Hit:92.10
LRU Hit:91.10, 2Q Hit:91.10
LRU Hit:92.70, 2Q Hit:92.70
LRU Hit:92.40, 2Q Hit:92.40
LRU Hit:91.80, 2Q Hit:91.80
LRU Hit:93.30, 2Q Hit:93.30
LRU Hit:92.10, 2Q Hit:92.10
LRU Hit:92.50, 2Q Hit:92.50
LRU Hit:91.00, 2Q Hit:91.00
LRU Hit:91.50, 2Q Hit:91.50
LRU Hit:92.30, 2Q Hit:92.30
LRU Hit:91.80, 2Q Hit:91.80
LRU Hit:91.60, 2Q Hit:91.60
LRU Hit:91.50, 2Q Hit:91.50
LRU Hit:92.30, 2Q Hit:92.30
LRU Hit:0.00, 2Q Hit:91.40
LRU Hit:0.00, 2Q Hit:89.30
LRU Hit:0.00, 2Q Hit:91.70
LRU Hit:13.10, 2Q Hit:88.90
LRU Hit:12.10, 2Q Hit:88.00
LRU Hit:11.20, 2Q Hit:90.00
LRU Hit:12.00, 2Q Hit:90.90
LRU Hit:11.60, 2Q Hit:89.60
LRU Hit:11.90, 2Q Hit:90.20
LRU Hit:12.30, 2Q Hit:89.20
LRU Hit:92.80, 2Q Hit:92.80
```